src/mouse_pos.rs

```diff
@@ -147,7 +147,7 @@ fn update_pos_ortho(
             .get(camera)
             .expect("only orthographic cameras are supported");
         let offset = Vec2::new(proj.left, proj.bottom);
-        world.0 = camera.mul_vec3((screen.0 + offset).extend(0.0)) * proj.scale;
+        world.0 = camera.mul_vec3(((screen.0 + offset) * proj.scale).extend(0.0));
```

Comment:
This could use a comment explaining the motivation behind the change. Otherwise it'll probably get reverted in the future.

Reply:
Tbh I'm not exactly sure why it works I just played around with it until it did :P
I don't know why multiplying by the scale before applying the transform fixes the issues I had but it did!